generators/generator-constants.js

```diff
@@ -112,12 +117,14 @@ const TEST_DIR = 'src/test/';
 // Note: this will be prepended with 'target/classes' for Maven, or with 'build/resources/main' for Gradle.
 const CLIENT_DIST_DIR = 'static/';
 
-const SUPPORTED_VALIDATION_RULES = ['required', 'unique', 'max', 'min', 'maxlength', 'minlength', 'maxbytes', 'minbytes', 'pattern'];
+const SUPPORTED_VALIDATION_RULES = Object.keys(validationOptions)
+    .map(key => validationOptions[key])
+    .filter(e => typeof e === 'string');
```

Comment:
We shouldn't have to do this (especially here).
There are other alternatives like updating the validationOptions' file.

Plus, if we do this here, what prevents us from doing it elsewhere (and duplicating the code)?

Reply:
Hi @MathieuAA : what is the exact issue? I just wanted to have all attributes of that class in there

Reply:
Oh okay. I guess the validations' file need refactoring. This isn't this PR's scope to do it, I'll write it down so that I can do it later.